packages/openapi-parser/src/utils/index.ts

```diff
@@ -1,26 +1,26 @@
-export * from './dereference'
-export * from './details'
-export * from './escapeJsonPointer'
-export * from './filter'
-export * from './getEntrypoint'
-export * from './getListOfReferences'
-export * from './getSegmentsFromPath'
-export * from './isFilesystem'
-export * from './isJson'
-export * from './isObject'
-export * from './isYaml'
-export * from './load'
-export * from './load/load'
-export * from './normalize'
-export * from './openapi'
-export * from './resolveReferences'
-export * from './toJson'
-export * from './toYaml'
-export * from './transformErrors'
-export * from './traverse'
-export * from './unescapeJsonPointer'
-export * from './upgrade'
-export * from './upgradeFromThreeToThreeOne'
-export * from './upgradeFromTwoToThree'
-export * from './validate'
-export * from './transform'
+export * from './dereference.ts'
```

Comment:
maybe out of scope for this PR, but we can get rid of this file too. Also the export *